landing-page/content/common/roadmap.md

```diff
@@ -30,33 +30,18 @@ Related design docs will be linked on the planning boards.
 # Priority 1
 
 * API: [Iceberg 1.0.0](https://github.com/apache/iceberg/projects/3) [medium]
-* Spark: [Merge-on-read plans](https://github.com/apache/iceberg/projects/11) [large]
-* Maintenance: [Delete file compaction](https://github.com/apache/iceberg/projects/10) [medium]
-* Flink: [Upgrade to 1.13.2](https://github.com/apache/iceberg/projects/12) (document compatibility) [medium]
 * Python: [Pythonic refactor](https://github.com/apache/iceberg/projects/7) [medium]
+* Spec: [Z-ordering / Space-filling curves](https://github.com/apache/iceberg/projects/16) [medium]
+* Spec: [Snapshot tagging and branching](https://github.com/apache/iceberg/projects/4) [small]
+* Views: [Spec](https://github.com/apache/iceberg/projects/6) [medium]
```

Comment:
```suggestion
* Views: [Spec](https://github.com/apache/iceberg/projects/6) [medium]
* Puffin: [Implement statistics information in table snapshot](https://github.com/apache/iceberg/pull/4741) [medium]
```

Reply:
I was going to suggest adding the word “additional” to not confuse people less familiar with the project - as statistics are somewhat integral to the Iceberg spec already - something like `Implement additional statistics information in table snapshot`.

But I see it came directly from the PR title so I think it’s fine. Eventually we should consult with Piotr for a more optimal high level description of the milestone but this is a good stand in for now (and possibly even for the medium term).